SPECS/strace/strace.spec

```diff
@@ -51,9 +56,10 @@ rm -rf %{buildroot}/*
 %{_mandir}/man1/*
 
 %changelog
-* Sat May 09 00:21:09 PST 2020 Nick Samson <nisamson@microsoft.com> - 5.1-2
-- Added %%license line automatically
-
+*   Wed Jan 06 2021 Andrew Phelps <anphel@microsoft.com> 5.1-3
+-   Patch tests with expected results. Increase test timeout.
```

Comment:
Wasn't `strace` one of the flaky tests, which passed or failed randomly? I'm asking to make sure, that updating expected values is the right solution.

Reply:
yes, these tests were inconsistent and not always failing. I updated the summary of this PR with the source of the test value changes. I'm expecting the test timeout increase to help with reliability as well